src/perl/Permabit/ModuleBase.pm

```diff
@@ -119,6 +121,13 @@ sub loadFromFiles {
     return 1;
   }
 
+  # If useUpstream module is set, we should have loaded from
+  # loadFromBinaryRPM.
+  if ($self->{useUpstream}) {
+    $log->debug("useUpstream flag set, loadFromBinaryRPM should have succeed");
+    return 0;
+  }
+
```

Comment:
Why did you add this block? It seems like it should be unnecessary. (i.e., if the load didn't work, we also won't find an SRPM for dm-vdo so the next step will also fail.)

Reply:
This block has been removed